{{ cookiecutter.repo_name }}/Makefile

```diff
@@ -36,45 +29,49 @@ clean:
 
 ## Lint using flake8
 lint:
-	flake8 src
-
-## Upload Data to S3
-sync_data_to_s3:
-ifeq (default,$(PROFILE))
-	aws s3 sync data/ s3://$(BUCKET)/data/
-else
-	aws s3 sync data/ s3://$(BUCKET)/data/ --profile $(PROFILE)
-endif
-
-## Download Data from S3
-sync_data_from_s3:
-ifeq (default,$(PROFILE))
-	aws s3 sync s3://$(BUCKET)/data/ data/
-else
-	aws s3 sync s3://$(BUCKET)/data/ data/ --profile $(PROFILE)
-endif
-
+	flake8 {{ cookiecutter.module_name }}
+
+{% if cookiecutter.dataset_storage.s3 %}
```

Comment:
should be `{%` if 's3' in cookiecutter.dataset_storage %}`

Reply:
Did you test this? These lines work for me as is.

Reply:
The error only happens when you chose none option in dataset_storage.